### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by MalteMagnussen** [Forceu/Gokapi#187] [Q&A] [answered]
Trying to set up Level 3 End-to-End Encryption: https://gokapi.readthedocs.io/en/latest/setup.html#encryption

But it just "hangs" when it gets to this point. 

![image](https://github.com/Forceu/Gokapi/assets/11192971/18bae5d5-1dca-455f-8374-baa4f0da6ae7)

Nothing in the log about it either.

**Comment by Forceu** [comment, marked as answer]
As mentioned in #188 it appears to be a problem with the included custom JavaScript
